src/Workspace/File/utils.tsx

```
@@ -147,3 +147,59 @@ export const generateUniqueId = (node: FileNode | GroupNode): string => {
+    document.body.removeChild(link);
+  } finally {
+    if (blobUrl) {
+      URL.revokeObjectURL(blobUrl);
+    }
+  }
+};
+
+/**
+ * 通用的默认分享处理函数
+ */
+export const handleDefaultShare = async (file: FileNode, locale?: any) => {
```

ESLint

[error] 'locale' is defined but never used.

(@typescript-eslint/no-unused-vars)